playground/hmr-ssr/__tests__/hmr-ssr.spec.ts

```diff
@@ -887,14 +887,19 @@ if (!isBuild) {
   test('assets HMR', async () => {
```

Comment:
The similar test in `hmr/__tests__/hmr.spec.ts` now tests HMR for inlined SVGs. Could we have a new test in `hmr` playground that tests non-inlined SVGs so that we cover these lines?
https://github.com/vitejs/vite/blob/d0399f792cb93ba2a7168903ef8a1323b9c65927/packages/vite/src/node/plugins/asset.ts#L183-L189

Reply:
Added 👍

Reply:
Thanks!